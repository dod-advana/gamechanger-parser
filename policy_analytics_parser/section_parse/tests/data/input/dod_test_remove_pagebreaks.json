{
    "filename": "DoDI 1341.12.pdf",
    "doc_type": "DoDI",
    "all_sections": [
        ["439 of Title 37, United States Code (U.S.C.). "],
        ["DoDI 1341.12, October 31, 2019 "],
        ["TABLE OF CONTENTS "],
        ["DoDI 1341.12, October 31, 2019 "],
        ["DoDI 5400.11 and DoD 5400.11-R. "],
        ["e.  Comply with the privacy procedures outlined in DoD Manual 6025.18, DoDI 1000.30, "],
        ["DoDI 1341.12"]
    ]
}
