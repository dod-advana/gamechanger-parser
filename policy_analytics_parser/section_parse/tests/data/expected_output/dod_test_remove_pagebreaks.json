{
    "all_sections": [
        ["439 of Title 37, United States Code (U.S.C.). "],
        ["TABLE OF CONTENTS "],
        ["DoDI 5400.11 and DoD 5400.11-R. "],
        ["e.  Comply with the privacy procedures outlined in DoD Manual 6025.18, DoDI 1000.30, "]
    ]
}
